sdk/core/azure-core/test/libcurl-stress-test/README.md

```diff
@@ -19,7 +19,7 @@ Where namespace will be created if missing , search directory can be any folder
 
 ATM the docker image is build by hand and hard-coded in the chart to simplify matters.  
 
-To build the image run "docker build -t <acr>/azuresdkforcpp/curlstress:v8  --build-arg targetTest=azure-core-libcurl-stress-test --build-arg build=on  ."
+To build the image run "docker build -t <acr>/azuresdkforcpp/curlstress:v8  --build-arg targetTest=azure-core-libcurl-stress-test --build-arg build=on  -f .\Dockerfile .\..\..\..\..\..\"
```

Comment:
Relative paths like this are very brittle. Can we make it more robust? Otherwise, please add a comment to clarify intent here.